### ISSUE TITLE
macOS file path is not relative

### ISSUE THREAD

**Issue by puhitaku** [puhitaku/r3build#3]
Directory tree:
```
.
├── docs
│   └── document.md
└── r3build.toml
```

r3build.toml:
```toml
[log]
all = true

[[job]]
type = "command"
when = "modified"
glob = "./docs/*.md"
command = "echo Hello"
```

On Linux, when I modify `./docs/document.md`:
```
R3BUILD Watcher    >> Accept: 'created' event on ./docs/4913
R3BUILD Watcher    >> Accept: 'modified' event on ./docs
R3BUILD Watcher    >> Accept: 'modified' event on ./docs/4913
R3BUILD Watcher    >> Accept: 'closed' event on ./docs/4913
R3BUILD Watcher    >> Ignore: (ratelimit) 'modified' event on ./docs
R3BUILD Watcher    >> Accept: 'deleted' event on ./docs/4913
R3BUILD Watcher    >> Ignore: (ratelimit) 'modified' event on ./docs
R3BUILD Watcher    >> Accept: 'moved' event on ./docs/document.md
R3BUILD Watcher    >> Ignore: (ratelimit) 'modified' event on ./docs
R3BUILD Watcher    >> Accept: 'created' event on ./docs/document.md
R3BUILD Watcher    >> Ignore: (ratelimit) 'modified' event on ./docs
R3BUILD Watcher    >> Accept: 'modified' event on ./docs/document.md
R3BUILD Watcher    >> Ignore: (ratelimit) 'modified' event on ./docs/document.md
R3BUILD Watcher    >> Accept: 'closed' event on ./docs/document.md
R3BUILD Watcher    >> Ignore: (ratelimit) 'modified' event on ./docs
R3BUILD Watcher    >> Accept: 'deleted' event on ./docs/document.md~
R3BUILD Watcher    >> Ignore: (ratelimit) 'modified' event on ./docs
R3BUILD noname     >> Ignore: (patterns don't match) 'created' event on ./docs/4913
R3BUILD noname     >> Ignore: (patterns don't match) 'modified' event on ./docs
R3BUILD noname     >> Ignore: (patterns don't match) 'modified' event on ./docs/4913
R3BUILD noname     >> Ignore: (patterns don't match) 'closed' event on ./docs/4913
R3BUILD noname     >> Ignore: (patterns don't match) 'deleted' event on ./docs/4913
R3BUILD noname     >> Ignore: (patterns don't match) 'moved' event on ./docs/document.md
R3BUILD noname     >> Ignore: (patterns don't match) 'created' event on ./docs/document.md
R3BUILD noname     >> Detected 'modified' event on ./docs/document.md
```

Also on macOS:
```
R3BUILD Watcher    >> Accept: 'created' event on /Users/takumi/dev/r3build-test/docs/4913
R3BUILD Watcher    >> Accept: 'modified' event on /Users/takumi/dev/r3build-test/docs
R3BUILD Watcher    >> Accept: 'deleted' event on /Users/takumi/dev/r3build-test/docs/4913
R3BUILD Watcher    >> Ignore: (ratelimit) 'modified' event on /Users/takumi/dev/r3build-test/docs
R3BUILD Watcher    >> Accept: 'moved' event on /Users/takumi/dev/r3build-test/docs/document.md
R3BUILD Watcher    >> Ignore: (ratelimit) 'modified' event on /Users/takumi/dev/r3build-test/docs
R3BUILD Watcher    >> Accept: 'created' event on /Users/takumi/dev/r3build-test/docs/document.md
R3BUILD Watcher    >> Ignore: (ratelimit) 'modified' event on /Users/takumi/dev/r3build-test/docs
R3BUILD Watcher    >> Accept: 'modified' event on /Users/takumi/dev/r3build-test/docs/document.md
R3BUILD Watcher    >> Ignore: (ratelimit) 'modified' event on /Users/takumi/dev/r3build-test/docs/document.md
R3BUILD Watcher    >> Accept: 'deleted' event on /Users/takumi/dev/r3build-test/docs/document.md~
R3BUILD Watcher    >> Ignore: (ratelimit) 'modified' event on /Users/takumi/dev/r3build-test/docs
R3BUILD noname     >> Ignore: (patterns don't match) 'created' event on /Users/takumi/dev/r3build-test/docs/4913
R3BUILD noname     >> Ignore: (patterns don't match) 'modified' event on /Users/takumi/dev/r3build-test/docs
R3BUILD noname     >> Ignore: (patterns don't match) 'deleted' event on /Users/takumi/dev/r3build-test/docs/4913
R3BUILD noname     >> Ignore: (patterns don't match) 'moved' event on /Users/takumi/dev/r3build-test/docs/document.md
R3BUILD noname     >> Ignore: (patterns don't match) 'created' event on /Users/takumi/dev/r3build-test/docs/document.md
R3BUILD noname     >> Ignore: (patterns don't match) 'modified' event on /Users/takumi/dev/r3build-test/docs/document.md
R3BUILD noname     >> Ignore: (patterns don't match) 'deleted' event on /Users/takumi/dev/r3build-test/docs/document.md~
```

**Comment by puhitaku** [comment]
It seems it originates from [this watchdog issue](https://github.com/gorakhargosh/watchdog/issues/735)

**Comment by puhitaku** [comment]
Better to solve this issue not by sending PR to watchdog but modifying incoming events?

**Comment by puhitaku** [comment]
Fixed in [Version 0.10.1](https://github.com/puhitaku/r3build/releases/tag/v0.10.1)
